test-scripts/launchTestUtilities.ts

```diff
@@ -33,8 +33,7 @@ export async function setupVSCodeTestInstance(): Promise<string> {
     console.log(`Setting up VS Code test instance, version: ${vsCodeVersion}`)
     const vsCodeExecutablePath = await downloadAndUnzipVSCode(vsCodeVersion)
     console.log(`VS Code test instance location: ${vsCodeExecutablePath}`)
-
-    await invokeVSCodeCli(vsCodeExecutablePath, ['--version'])
+    console.log(await invokeVSCodeCli(vsCodeExecutablePath, ['--version']))
```

Comment:
`invokeVSCodeCli` return a buffer, will the console.log statement be of any value? Same below.

Reply:
console.log prints the buffer contents